tensornetwork/network.py

```diff
@@ -845,6 +845,22 @@ def split_node(
     U * sqrt(S) and the right node's tensor will be sqrt(S) * (V*) where V* is
     the adjoint of V.
 
+    The singular value decomposition is truncated if `max_singular_values` or
+    `max_truncation_err`.
```

Comment:
... or `max_truncation_err` is not `None`.

Reply:
Oops, that got lost during an edit!